docker/validator-dev/build.sh

```diff
@@ -28,7 +28,7 @@ else
   docker run --name libra-node-builder-container -d -i -t -v $DIR/../..:/libra $PROXY_RUN libra-node-builder bash
 fi
 
-docker exec -i -t libra-node-builder-container bash -c 'source /root/.cargo/env; cargo build --release -p libra-node --target-dir /target && /bin/cp /target/release/libra-node target/libra-node-builder/'
+docker exec -i -t libra-node-builder-container bash -c 'source /root/.cargo/env; RUSTFLAGS="-Ctarget-cpu=sandybridge -Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3" cargo build --release -p libra-node --target-dir /target && /bin/cp /target/release/libra-node target/libra-node-builder/'
```

Comment:
is this exhaustive? there's probably more docker builds floating around no?

Reply:
oops, I didn't mean to add this in this commit. Yes there are more but not sure where it should be added exactly : o

Reply:
now added all of them in the last commit